### DISCUSSION TITLE
Freeform Dashbaord Error Messages - asking for email template that is no longer required

### DISCUSSION THREAD

**Discussion by acantes** [solspace/craft-freeform#140] [Questions & Answers] [answered]
Hi, I am seeing errors in the control panel Freeform Dashboard screen every time any of the three forms on my client's website are submitted. They are all similar to this:

[2021-04-29T20:53:22.850100+01:00] Email Notification.WARNING: Email notification template with ID arrange-viewing-customer.html not found {"form":"Arrange Viewing Form"} []

The forms originally had a customer email notification template set to send to the email address they provided in the form, but we have since removed the link to the email field setup in the Freeform form. The email template configuration dropdown has been saved to "Select a template" for the form.

The errors still persist even though there is no link anywhere I can see to the customer email template. 

Admin recipients receive a separate email notification correctly.

Happy to provide any further information you may require.

Craft CMS v3.6.12
Freeform Pro v3.11.2

**Comment by kjmartens** [comment, marked as answer]
@acantes it is very likely the issue has something to do with am orphaned form looking for that deleted email notification template. If I had to guess (and judging by your wording), it would likely be happening in an Email field type. 

To resolve this issue:

1. Open the form inside the form builder
2. Click on the Email field that is affected
3. In the property editor (right column), temporarily select ANY other notification template for it.
4. Click the Save button to save the form. Then refresh the page in the browser.
5. Click on the Email field affected again (if not already open in the property editor).
6. Change the email notification template dropdown back to "Select a template" (assuming you want no notification to happen).
7. Save the form.

Try submitting the form again and see if new errors are logged in the Freeform error log.

You may need to repeat those steps for other affected forms. :)

**Comment by acantes** [comment]
Thanks for the quick response Kelsey, that fixed it for me. The Solspace equivalent of turning it off and back on again!
